diff --git a/dist/626.js b/dist/626.js
index 40f91ea025e88558fbd6ca3823a8f8bde062ebe2..f78a4ffb6ef09bb5a216f17596cced3bc1433af2 100644
--- a/dist/626.js
+++ b/dist/626.js
@@ -1,6 +1,6 @@
 /*! LICENSE: 626.js.LICENSE.txt */
 let runtimeProcessArgs, runtimeInfo, swcHelpersPath, pluginHelper_htmlPlugin;
-import { rspack as core_rspack } from "@rspack/core";
+import { rspack as core_rspack } from "@rspack-canary/core";
 import node_util, { stripVTControlCharacters } from "node:util";
 import node_path, { dirname as external_node_path_dirname, isAbsolute as external_node_path_isAbsolute, join, posix, relative, resolve as external_node_path_resolve, sep, win32 } from "node:path";
 import { builtinModules, createRequire, createRequire as __rspack_createRequire } from "node:module";
